share/dkg/pedersen/dkg.go

```diff
@@ -72,7 +72,7 @@ type Config struct {
 	// `vss.MinimumT(len(NewNodes))`. This threshold indicates the degree of the
 	// polynomials used to create the shares, and the minimum number of
 	// verification required for each deal.
-	Threshold int
+	Threshold int64
```

Comment:
I don't think a Threshold larger than 2^32 makes sense anyway, why `int64` here?.
This looks more like a `uint32`